jetty-ee10/jetty-ee10-proxy/src/main/java/org/eclipse/jetty/ee10/proxy/AsyncMiddleManServlet.java

```diff
@@ -858,15 +872,12 @@ public void write(ByteBuffer buffer, Callback callback)
         }
     }
 
-    private static class Chunk
+    private record Chunk(ByteBuffer buffer, Callback callback)
```

Comment:
`Chunk` is quite a loaded term now, maybe this record should be renamed to something like `BufferWithCallback` or something to avoid confusion with `Content.Chunk`?